addons/l10n_in/views/res_config_settings_views.xml

```diff
@@ -29,6 +29,12 @@
                     </setting>
                 </block>
             </app>
+            <block id="print_vendor_checks_setting_container" position="inside">
+                <setting id="vendor_payment_order_file" company_dependent="1"
+                    help="Generate Vendor Payment Order file(csv file), upload to your bank to make the payments" invisible="country_code != 'IN'">
```

Comment:
Since we add a help here, could you export the pot again :smile:

Reply:
Updated the pot according to your suggestion